pkgs/applications/misc/dmenu-rs/default.nix

```diff
@@ -19,13 +19,13 @@
 # See: https://github.com/Shizcow/dmenu-rs#plugins
 stdenv.mkDerivation rec {
   pname = "dmenu-rs";
-  version = "5.5.2";
+  version = "5.5.3";
 
   src = fetchFromGitHub {
     owner = "Shizcow";
     repo = pname;
```

Comment:
```suggestion
    repo = "dmenu-rs";
```
#277994

Reply:
Thanks for including the justification! Committing ...